perllib/FixMyStreet/DB/Result/User.pm

```diff
@@ -210,6 +210,21 @@ sub body {
     return $self->from_body->name;
 }
 
+=head2 area_ids
+
+Returns the areas a user has been assigned to as an array
+
+=cut
+
+sub area_ids {
+    my $self = shift;
+    return [] if !defined $self->areas;
+    my @areas = split(/,/, $self->areas);
+    @areas = grep { $_ ne '' } @areas;
+    @areas = map { $_ + 0 } @areas;
```

Comment:
Is there anywhere that needs these to be ints, apart from the tests? Probably best to not use coercion and let Perl figure it out.